### ISSUE TITLE
Connect to Apple TV

### ISSUE THREAD

**Issue by AlwinGit** [T-vK/ESP32-BLE-Keyboard#44]
The ESP doesn't show up in the Apple TV when it is searching for new bluetooth keyboards.

**Comment by T-vK** [comment]
Yeah, there have been plenty of reports with apple devices rejecting it. Since I don't own any Apple devices there's not much I can do.

**Comment by AlwinGit** [comment]
Is there a way that I can help?

**Comment by T-vK** [comment]
Depeneds on how much work and effort you want to put into it. You could go through the existing issues and try the workarounds or fixes people have posted for other apple devices.

**Comment by T-vK** [comment]
You could try #111 it may fix your issues.
